core/commands/ping.go

```diff
@@ -182,9 +182,11 @@ func pingPeer(ctx context.Context, n *core.IpfsNode, pid peer.ID, numPings int)
 }
 
 func ParsePeerParam(text string) (ma.Multiaddr, peer.ID, error) {
-	// to be replaced with just multiaddr parsing, once ptp is a multiaddr protocol
-	idx := strings.LastIndex(text, "/")
+	idx := strings.LastIndex(text, "/ipfs/")
```

Comment:
There's a few instances of the literal string `"/ipfs/"` in the package, would it be better to declare it as a named const?
e.g. `peerPrefix` or something

Reply:
Ideally we'd parse those as a multiaddr, but for that we'd need to define new multiaddr proto for that.